src/api/clinical/medication/drug/drug.validator.ts

```diff
@@ -81,6 +82,7 @@ export class DrugValidator extends BaseValidator{
 
         var filters: DrugSearchFilters = {
             Name : request.query.name ?? null,
+            genericName : request.query.genericName ?? null,
```

Comment:
genericName : request.query.genericName ?? null,
G-should be capital on the left hand side.